### PR TITLE
Documentation workflow

### PR DESCRIPTION
**Pull request by MaxDiz**
initial draft for review/discussion. Once we have an agreed upon draft to publish, we may want to move this section to a separate page within the doc site hierarchy. 

Also includes a couple of formatting improvements to the Development Workflow (noticed as I was reviewing).

### CODE REVIEW

File: content/en/core/process/workflow.md [hunk 1]
```diff
@@ -140,9 +140,56 @@ Regardless of what you do with the issue, please:
...
  
-# Workflow for contributing to cht-docs
-Unlike the above development workflow, the workflow for contributing to [cht-docs](https://github.com/medic/cht-docs) does not involve release management and Acceptance Testing. In order to streamline the process of contributing to docs while still maintaining quality, please submit a pull request with any changes to [cht-docs](https://github.com/medic/cht-docs). One of the repository's maintainers will review the PR, request additional changes as needed, and merge the PR when it is ready. 
+# Documentation Workflow
 
-Where possible, please follow our [docs style guide]({{< ref "docs-style-guide" >}}).
+This documentation site (and related [GitHub site](https://github.com/medic/medic.github.io)) is the CHT’s main resource for informing users on the product. It is designed to provide Community Health Toolkit users clear information on product functionality, as well as the necessary guidelines to accomplish key steps for setting up, deploying, and maintaining the product.
```
[the remark this answers is not in the document]
Comment (abbyad) [reply content/en/core/process/workflow.md:145]: Would be good to make the intro paragraph a little more crisp to make it clearer the purpose and content of the docs site. Here is an attempt -- feel free to hash up as you see fit.

```suggestion
This Community Health Toolkit documentation site is the primary resource for partners, app designers, and developers looking to learn about, as well as to build digital health apps using the CHT. The documentation site includes product feature overviews, design resources, implementer guides, and technical reference documentation. 
```

File: content/en/core/process/workflow.md [hunk 2]
```diff
@@ -140,9 +140,56 @@ Regardless of what you do with the issue, please:
...
-Unlike the above development workflow, the workflow for contributing to [cht-docs](https://github.com/medic/cht-docs) does not involve release management and Acceptance Testing. In order to streamline the process of contributing to docs while still maintaining quality, please submit a pull request with any changes to [cht-docs](https://github.com/medic/cht-docs). One of the repository's maintainers will review the PR, request additional changes as needed, and merge the PR when it is ready. 
+# Documentation Workflow
 
-Where possible, please follow our [docs style guide]({{< ref "docs-style-guide" >}}).
+This documentation site (and related [GitHub site](https://github.com/medic/medic.github.io)) is the CHT’s main resource for informing users on the product. It is designed to provide Community Health Toolkit users clear information on product functionality, as well as the necessary guidelines to accomplish key steps for setting up, deploying, and maintaining the product. 
 
+Documentation is a collaborative effort among all Community members. To improve information on the documentation site, we encourage commenting, asking questions, and sharing ideas for improvement.
```
Comment (abbyad) [inline content/en/core/process/workflow.md:147]: It is great to call out all community members to collaborate, although it is not clear from this line how they would do it. Would be helpful here to mention that they could contribute edits too. 

Can you reword this to serve as a lead-in to the rest of the docs workflow sections?

File: content/en/core/process/workflow.md [hunk 3]
```diff
@@ -140,9 +140,56 @@ Regardless of what you do with the issue, please:
...
+
+### Feature Education
+Mainly prepared for a non-technical or semi-technical audience, it focuses on the [core features of the CHT]({{< ref "apps/features" >}}). Feature education includes summaries of feature functionality and screenshots with annotations or explanations of specific parts of the workflow as needed for added clarity. 
+
+### Technical Documentation
+Mainly prepared for a technical audience, it describes the [CHT Core Framework]({{< ref "core" >}}) and its [reference components]({{< ref "apps/reference" >}}). It includes information on technical implementation, design decisions, architecture descriptions, schemas, and source code. 
+
+### How-to Guides and Tutorials
+Mainly prepared for implementing partner capacity building, it includes best practices [quick guides]({{< ref "apps/guides" >}}) and [tutorials]({{< ref "apps/tutorials" >}}) on functional implementation of a feature (or group of features) and system configuration to assist users in completing specific objectives. 
+
```
Comment (abbyad) [inline content/en/core/process/workflow.md:161]: What is the purpose of explaining this here? An orientation to the content would help - so perhaps present the structure/organization of content first, and then explain the intended audiences, rather than the other way around.

File: content/en/core/process/workflow.md [hunk 4]
```diff
@@ -140,9 +140,56 @@ Regardless of what you do with the issue, please:
...
+## Issues
+
+All documentation and technical documentation site issues are managed in [Github cht-docs](https://github.com/medic/cht-docs/issues). When creating issues, add the appropriate [labels](https://github.com/medic/cht-docs/labels) to help provide guidance to other community members.
```
Comment (abbyad) [inline content/en/core/process/workflow.md:164]: Are there minimum required labels? Also, just a note that not all GH users will have access to adding labels. As part of updates to this workflow we should consider what doc site issue templates we want, and start fleshing those out.

Comment (abbyad) [inline content/en/core/process/workflow.md:164]: Rewording first sentence in an attempt to make it easier to read.

```suggestion
All documentation and technical issues for the documentation site are managed in the [`cht-docs` repo on GitHub](https://github.com/medic/cht-docs/issues). When creating issues, add the appropriate [labels](https://github.com/medic/cht-docs/labels) to help provide guidance to other community members.
```

File: content/en/core/process/workflow.md [hunk 5]
```diff
@@ -140,9 +140,56 @@ Regardless of what you do with the issue, please:
...
+## Prioritization
+
+CHT documentation is treated similar to CHT code: the aim is to improve it as often as possible. Documentation is prioritized in three different ways:
+
+1. Feature education is completed at the time of releasing the related issue in GitHub. At the beginning of each release, an issue(s) is created, it is added to the release project, and a lead is assigned. 
+
+2. Technical documentation is completed at the time of closing the related issue in a GitHub release. 
+
+3. How-to guides and tutorials are prioritized alongside other product work as part of the roadmap planning process.
+
```
Comment (abbyad) [inline content/en/core/process/workflow.md:175]: I don't think this section is necessary, as the requirements for documentation should be listed in the dev workflow. You could include a not about when docs are introduced in the doc structure overview above. For example: "Reference documentation is updated as part of GH issues in CHT Core", "New guides are developed as part of the release materials when a new feature is introduced".

File: content/en/core/process/workflow.md [hunk 6]
```diff
@@ -140,9 +140,56 @@ Regardless of what you do with the issue, please:
...
+## Writing
+
+We place a high importance on consistency and usability of CHT documentation. After all, the CHT is an open source software deployed in communities around the globe. The CHT [documentation style guide]({{< ref "docs-style-guide" >}}) will help you to write documentation in the most consistent and useful way.
```
Comment (abbyad) [inline content/en/core/process/workflow.md:178]:
```suggestion
We place a high importance on consistency and usability of CHT documentation so that it is accessible and understood by a wide audience. The CHT [documentation style guide]({{< ref "docs-style-guide" >}}) will help to write documentation in the most consistent and useful way.
```

File: content/en/core/process/workflow.md [hunk 7]
```diff
@@ -140,9 +140,56 @@ Regardless of what you do with the issue, please:
...
+{{% alert title="Note" %}} Every commit message should be able to complete the following sentence:
+When applied, this commit will: {YOUR COMMIT MESSAGE} {{% /alert %}} 
+
+## Reviewing
+
+The documentation author should assign a reviewer for any documentation changes that are made. We use the following review structure:
+
+* Features education is reviewed by a Product Manager and a Community Manager 
+* Technical documentation is reviewed by an Engineer
+* How-to guides and tutorials are reviewed by a Capacity Building Manager and a Community Manager
```
Comment (abbyad) [inline content/en/core/process/workflow.md:195]: Seems to be missing design system. That said, I think we are better off having this automated by CODEOWNERS mechanism instead of leaving it up to folks to figure out who the reviewer should be. We should instead give instruction to the review as to what they should do, and who should merge. We can work out that language together.

File: content/en/core/process/workflow.md (hunk 5 above)
Comment (MaxDiz) [reply content/en/core/process/workflow.md]: Good point. I deleted this section

File: content/en/core/process/workflow.md (hunk 4 above)
Comment (MaxDiz) [reply content/en/core/process/workflow.md]: changed this section substantially

File: content/en/core/process/workflow.md (hunk 3 above)
Comment (MaxDiz) [reply content/en/core/process/workflow.md]: Updated this section to focus on "content types" and also added a section to review documentation site organization

File: content/en/core/process/workflow.md (hunk 2 above)
Comment (MaxDiz) [reply content/en/core/process/workflow.md]: updated and included a "getting started" section

File: content/en/core/process/workflow.md [hunk 8]
```diff
@@ -140,9 +140,85 @@ Regardless of what you do with the issue, please:
...
+
+Mainly prepared for a non-technical audience, these content types focus on application features and concepts that effect deployment of the CHT. It includes summaries of basic concepts for building the CHT, as well as feature functionality and screenshots with explanations of specific parts of the workflow.
+
+### Examples, Guides, and Tutorials
+
+Mainly prepared for implementing partner capacity building, these content types include best practices on functional implementation of a feature (or group of features), care workflows, and system configuration to assist users in completing specific objectives. 
+
+### Reference, Architecture, and Data Management
+
+Mainly prepared for a technical audience, these content types describe the CHT Core Framework and its reference components. It includes information on technical implementation, design decisions, architecture descriptions, schemas, and source code.
```
Comment (abbyad) [inline content/en/core/process/workflow.md:223]: I'd recommend removing these two sections for now. We can discuss separately what would be useful to include, and do so in a follow-up PR.

```suggestion
```

**Comment by MaxDiz** [comment]
This looks good to me and can be merged

File: content/en/core/process/workflow.md [hunk 9]
```diff
@@ -140,9 +140,57 @@ Regardless of what you do with the issue, please:
...
+When applied, this commit will: {YOUR COMMIT MESSAGE} {{% /alert %}} 
+
+## Reviewing Pull Requests
+
+Documentation reviewers are automated by the [code owners](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners) mechanism. In general, reviewers should:
+
+1. Read the PR description to understand the changes made, as well as any linked issues
+2. Review any comments by other reviewers
+3. Select the **Files changed** tab to see the files and lines changed
+4. Click the + symbol beside the line you want to comment on. To select multiple lines at once, hold shift then click and drag across multiple lines.
```
Comment (abbyad) [inline content/en/core/process/workflow.md:187]: Just tested and shift is not needed. Just click and drag.

```suggestion
4. Click the **+** beside the line you want to comment on. To select multiple lines at once, click the **+** of the top line of the selection, drag down to the bottom line, and release.
```

File: content/en/core/process/workflow.md [hunk 10]
```diff
@@ -140,9 +140,57 @@ Regardless of what you do with the issue, please:
...
+### Contributing Basics
+
+* It is helpful to be comfortable with [git]({{< ref "core/guides/using-git" >}}) and [GitHub](https://lab.github.com/) to contribute to the CHT community.
+* The documentation source is in [GitHub](https://github.com/medic/cht-docs). The content pages are in the `/content/en/` directory.
+* Documentation is written in [Markdown](https://www.markdownguide.org/). 
+* The CHT site build uses [Hugo](https://gohugo.io/). You can also setup a [local clone](https://github.com/medic/cht-docs/blob/master/README.md#installing-hugo). 
+
+## Writing Documentation
+
+We place a high importance on consistency and usability of CHT documentation so that it is accessible and understood by a wide audience. The CHT [documentation style guide]({{< ref "docs-style-guide" >}}) will help to write documentation in the most consistent and useful way.
```
Comment (abbyad) [inline content/en/core/process/workflow.md:169]: [Who is "we" here](https://docs.communityhealthtoolkit.org/docs-style-guide/#using-we)?

```suggestion
A high degree of importance is put on consistency and usability of CHT documentation so that it is accessible and understood by a wide audience. The CHT [documentation style guide]({{< ref "docs-style-guide" >}}) will help to write documentation in the most consistent and useful way.
```

File: content/en/core/process/workflow.md [hunk 11]
```diff
@@ -140,9 +140,57 @@ Regardless of what you do with the issue, please:
...
+
+## Commits to GitHub
+
+The main branch is `master` which must be kept stable since it is deployed to the doc site. All documentation changes should be done in a branch with a Pull Request when ready for review. This means that a maintainer has signed off on the change before it hits the master branch.
+
+Format your commit messages according to the Git convention where the first line should be a short title/summary (50 characters or so) with more details in a separate paragraph (if needed).
+
+{{% alert title="Note" %}} Every commit message should be able to complete the following sentence:
+When applied, this commit will: {YOUR COMMIT MESSAGE} {{% /alert %}} 
+
```
Comment (abbyad) [inline content/en/core/process/workflow.md:179]: Would be good to add a line here about creating a PR.

Comment (abbyad) [reply content/en/core/process/workflow.md:179]: We should add something about creating the PR, setting the review, and label if appropriate.

```suggestion

When your branch is ready for review, create a [Pull Request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request). If you know who you'd like to review the PR, you can assign them directly. If you are unsure, you can leave it to the maintainers to handle the PR. 

{{% alert title="Note" %}}If the PR is part of an open issue in cht-core, add the [`Blocked: waiting on AT` label](https://github.com/medic/cht-docs/labels/Blocked%3A%20waiting%20on%20AT) so that the PR isn't accidentally merged prematurely, before the issue is acceptance tested and complete.{{% /alert %}} 

```
